### ISSUE TITLE
No error on clean install

### ISSUE THREAD

**Issue by tnViking** [eslint/eslint#4358]
I just did a clean install of node and eslint.
Tried to run eslint file.js, but no errors were given, nor anything really. Looked at docs that you need to run ```eslint --init```.
This should maybe be given as an error if the init is missing

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by tnViking** [comment]
This happened to any versionsa above 0.24.1

**Comment by ilyavolodin** [comment]
After version 0.24.1 we removed all of the default rules from the distribution. You should either manually create .eslintrc file or run `eslint -init` to automatically create one for your repository.
